package.json

```diff
@@ -366,6 +366,11 @@
           "type": "boolean",
           "default": true,
           "description": "Specifes whether OmniSharp should use VS Code editor settings for C# code formatting (use of tabs, indentation size)."
+        },
+        "omnisharp.format.enable": {
```

Comment:
Bad idea about "csharp." prefix because each non-builtin plugin should use own namespace prefix for custom settings on it. For example, vscode has builtin typescript support - its why there are "typescript.XXX" options, specified for builtin language. What does it means - "csharp.formatting.enable"? Is it for omnisharp plugin, or any other? Who knows, i suggest to move all options to "omnisharp.XXX" namespace.